en/docs/get-started/tutorials/create-your-first-rest-api.md

```diff
@@ -27,273 +26,60 @@ In this section, let's develop the application that retrieves COVID-19 related s
     | **Name**        | `COVID-19Stats`                       |
     | **Description** | `Get statistics relating to COVID-19` |
 
-4. Click **Create**.
-
-    This takes you to the **Components** page shown below.
-
-    ![Create component](../../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-full}
+4. Click **Create**. This takes you to the **Components** page.
 
 ### Step 1.2: Add a REST API component
 
 Let's create a new REST API component as follows:
 
-1. On the **Components** page, click **Create**.
+1. On the **Components** page, click **+Create** on the the **REST API** card.
```

Comment:
Suggestion: "On the **Components** page, click **+Create** under **REST API**.

Reply:
it's not a section.. there's a card

Reply:
changed +Create to Create